include/deal.II/base/partitioner.h

```diff
@@ -561,13 +561,13 @@ namespace Utilities
       IndexSet ghost_indices_data;
 
       /**
-       * Caches the number of ghost indices. It would be expensive to use @p
+       * A variable that caches the number of ghost indices. It would be expensive to use @p
```

Comment:
I think writing `A variable caching the number...` would sound better (same for `contains`, `stores`, `holds`, etc. below). What do you think?

Reply:
If you think so -- you're the native speaker, I'm not. I'm happy to defer to your sense of what sounds better.